### PR TITLE
chore: remove deprecated font (regular)

### PR DESCRIPTION
**Pull request by kathaypacific**
### Description

This PR replaces all `fontStyles.regular` with `typeScale.bodyMedium`. The difference is +2pt on lineHeight.

### Test plan

n/a

### Related issues

- Fixes RET-1174

### Backwards compatibility

Y

### Network scalability

If a new NetworkId and/or Network are added in the future, the changes in this PR will:

- [ ] Continue to work without code changes, OR trigger a compilation error (guaranteeing we find it when a new network is added)

### CODE REVIEW

File: src/components/Carousel.tsx
Comment (jophish) [inline src/components/Carousel.tsx:1]: Why delete this file? Is this related somehow?

File: src/styles/fonts.tsx [hunk 1]
```diff
@@ -148,14 +120,3 @@ export const typeScale = StyleSheet.create({
     letterSpacing: 0.2,
   },
 })
-
-/***
- * @deprecated
- * Use typeScale instead
- */
-export const fontStyles = StyleSheet.create({
```
Comment (jophish) [inline src/styles/fonts.tsx:156]: :tada:

**Review by jophish — APPROVED** [review]

File: src/components/Carousel.tsx
Comment (kathaypacific) [reply src/components/Carousel.tsx:1]: yes i was supposed to update the font style in it and then found it wasn't being used, so deleted it rather than make the update
